.github/workflows/R-CMD-check.yaml

```diff
@@ -57,5 +60,5 @@ jobs:
             secret = '${{ secrets.SHINYAPPS_SECRET }}'
           )
       - name: Deploy
-        run: |
+        if: ${{ steps.examples.deploy == 'true' }}        run: |
```

Comment:
Undo this change por favor

Reply:
lol I'm too fast 6fccc70be96f39203fbffba0db4b173b7a5d4064